src/jsonrpc/miner_jsonrpc_sc.erl

```diff
@@ -9,15 +9,15 @@
 handle_rpc(<<"sc_active">>, []) ->
     case (catch blockchain_state_channels_server:active_sc_ids()) of
         {'EXIT', _} -> ?jsonrpc_error(timeout);
-        undefined -> #{<<"active">> => <<"none">>};
+        undefined -> #{<<"active">> => null};
```

Comment:
Use `null` where appropriate.

Reply:
Would it be more appropriate to return an empty list here? (and anywhere else where a list is returned)?